admin_guide/cluster_management.rst

```diff
@@ -193,6 +193,58 @@ The coordinator node needs to know roles' passwords in order to communicate with
 
 Sometimes workers need to connect to one another, such as during :ref:`repartition joins <repartition_joins>`. Thus each worker node requires a copy of the .pgpass file as well.
 
+.. _phone_home:
+
+Checks For Updates and Cluster Statistics
+#########################################
+
+Unless you opt out, Citus checks if there is a newer version of itself during installation and every twenty-four hours thereafter. If a new version is available, Citus emits a notice to the database logs:
+
+.. code-block::
+
+  a new minor release of Citus (X.Y.Z) is available
```

Comment:
I don't see the code block including the notice in the [deployment](https://citus-doc-preview-pr-497.herokuapp.com/admin_guide/cluster_management.html#checks-for-updates-and-cluster-statistics).